main.py

```
@@ -13,6 +13,23 @@
 import vertexai
 from vertexai.generative_models import GenerativeModel
 import json
+import google.generativeai as genai
+import requests
```

Ruff (0.12.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)